main.py

```diff
@@ -0,0 +1,21 @@
+from environment import Environment, preprocess_data
```

Comment:
should we keep our model execution within each model's class, or have a single main class to run every model?

Reply:
I think both is fine if you want to directly run the model file.